main/main.c

```
@@ -17,7 +25,13 @@ static void wifi_event_handler(void *arg, esp_event_base_t event_base,
     } else if (event_base == IP_EVENT && event_id == IP_EVENT_STA_GOT_IP) {
         ip_event_got_ip_t *event = (ip_event_got_ip_t *)event_data;
         ESP_LOGI(TAG, "Got IP: " IPSTR, IP2STR(&event->ip_info.ip));
-        // TODO: Start WebSocket server here (Phase 2)
+        esp_err_t ret = ws_server_init(&g_ws_server, 4869, on_ws_message);
+        if (ret == ESP_OK) {
+            ESP_LOGI(TAG, "Relay listening on ws://" IPSTR ":4869",
```

Cppcheck (2.19.0)

[error] There is an unknown macro here somewhere. Configuration is required. If IPSTR is a macro then please configure it.

(unknownMacro)